packages/angular/package.json

```diff
@@ -38,7 +39,7 @@
     "@angular/router": "~10.2.5",
     "ng-packagr": "^10.1.0",
     "rxjs": "6.5.5",
-    "typescript": "~4.0.2",
+    "typescript": "4.0.2",
```

Comment:
l/q: Why pin the version here and remove it in `angular-ivy`? Should we also remove it here and revert if problematic?

Reply:
actually it does not work here with 4.9 😅 some type issue.

Reply:
For reference, this is the error we get:

```
ERROR: The Angular Compiler requires TypeScript >=3.9.2 and <4.1.0 but 4.9.5 was found instead.
An unhandled exception occurred: The Angular Compiler requires TypeScript >=3.9.2 and <4.1.0 but 4.9.5 was found instead.
```

So I guess this is not supported there!

Reply:
Yes that makes sense to me now, I just didn't think of this. Let's leave it as is!